pxb/jenkins/checkout

```diff
@@ -69,5 +73,9 @@ if [ "$SOURCE_NAME" == 'PXB24' -o "$SOURCE_NAME" == 'ALL' ]; then
         if [ -n "${PXB24_REPO}" -a -n "${PXB24_BRANCH}" ]; then
             git pull origin ${PXB24_BRANCH}
         fi
+
+        git submodule deinit -f . || true
+        git submodule init
+        git submodule update
```

Comment:
why not just `git submodule update --init`

Reply:
@EvgeniyPatlan @satya-bodapati , I did this way because it is done so in the rest of the configurations of the pipelines. I guess the motives might be the the syntax you've proposed is not supported by new versions of git. https://stackoverflow.com/a/10168693/1449471

Reply:
@EvgeniyPatlan what is the git version used in pipelines?

Reply:
@satya-bodapati we always install latest available in repos